tests/Aspire.Confluent.Kafka.Tests/ConsumerConfigurationTests.cs

```diff
@@ -102,14 +102,96 @@ public void ConnectionNameWinsOverConfigSection(bool useKeyed)
 
         using var host = builder.Build();
         var connectionFactory = useKeyed
-            ? host.Services.GetRequiredKeyedService(ReflectionHelpers.ConsumerConnectionFactoryStringKeyStringValueType.Value!, "messaging")
-            : host.Services.GetRequiredService(ReflectionHelpers.ConsumerConnectionFactoryStringKeyStringValueType.Value!);
+            ? host.Services.GetRequiredKeyedService(ReflectionHelpers.ConsumerConnectionFactoryStringKeyStringValueType.Value, "messaging")
+            : host.Services.GetRequiredService(ReflectionHelpers.ConsumerConnectionFactoryStringKeyStringValueType.Value);
 
-        ConsumerConfig config = GetConsumerConfig(connectionFactory)!;
+        var config = GetConsumerConfig(connectionFactory)!;
 
         Assert.Equal(CommonHelpers.TestingEndpoint, config.BootstrapServers);
     }
 
+    [ConditionalTheory]
```

Comment:
Why are these `ConditionalTheory`? I don't think they use SkipExceptions.

```suggestion
    [Theory]
```

Reply:
done for all UT in that file. Others were not concerned